internal/environment/configure_selection.go

```diff
@@ -11,18 +12,18 @@ import (
 
 func ConfigureSelection(cat *catalog.Catalog, configFilePath string, all bool) error {
 	// Load fresh copy of config file, without any alterations/defaults applied
-	cfg, err := config.LoadFile(configFilePath)
+	cfg, err := config.Load(filepath.Dir(configFilePath), "")
```

Comment:
Just want to make sure this load func does not merge in anything else than the bare config file itself, as we don't want to persist back anything that was not in the originally loaded file. Have you tested it manually just to sanity check? That's the most important thing that popped to my mind.

Reply:
This loads a fresh copy of the entire config object, and modifies and saves the user config back separately from everything else, I believe it to be safe!

Reply:
I'll make it nicer though